_posts/aws/2019-06-03-AWS-api-gateway-integration.md

```diff
@@ -0,0 +1,83 @@
+---
+date: 2019-05-29
+title: AWS API Gateway Integration
+categories:
+   - AWS
+description: How to integrate AWS API Gateway to expose Spinnaker webhook
```

Comment:
API Gateway not only is useful for webhooks, but for exposing any Gate endpoint in general. For example this can also be useful for GitHub triggers. Instead of `to expose Spinnaker webhook` it could be rephrased to `to expose Spinnaker API endpoints (Gate)`